rlang/pom.xml

```diff
@@ -109,6 +109,13 @@
             <version>${spark.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-client</artifactId>
+            <version>2.7.7</version>
```

Comment:
@zjffdu Why you are using a hard coded version? Maybe it's better use `hadoop.version`? (https://github.com/apache/zeppelin/blob/master/pom.xml#L146)